modules/representation_management/spec/support/rswag_config.rb

```diff
@@ -43,7 +43,9 @@ def schemas
       ErrorModel: error_model,
       Errors: errors,
       Error: error,
-      PowerOfAttorneyResponse: power_of_attorney_response
+      PowerOfAttorneyResponse: power_of_attorney_response,
+      VeteranServiceRepresentative: veteran_service_representative_schema,
+      VeteranServiceOrganization: veteran_service_organization_schema
```

Comment:
Can you find out what case these keys should be? I don't know swagger that well so I don't know. If it really doesn't matter (you said you weren't sure if it does) can you please choose a case rather than there being a mix and update the file accordingly?

Currently there's snake, pascal and camel being used in this file.

**Actually I just found docs/setup/rswag_setup.md which just uses camel. Probably best to go with that.

Reply:
All the schema names are now camelCased.

Reply:
The screenshot above is updated as well.